.github/workflows/release.yml

```diff
@@ -0,0 +1,130 @@
+name: Release
+
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  release-build:
+    runs-on: ubuntu-20.04
+    env:
+      # TODO
+      RTE_CONTAINER_IMAGE: quay.io/k8stopologyawarewg/resource-topology-exporter:ci
+    steps:
+    - name: checkout sources
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: setup golang
+      uses: actions/setup-go@v2
+      id: go
+      with:
+        go-version: 1.16
+
+    - name: verify modules
+      run: go mod verify
+
+    - name: set release version env var
+      run: |
+        echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+
+    - name: build test binary
```

Comment:
Here the execution code is the same as the gh ```e2e.yml``` 
Maybe we can write the ci test as a separate action and then we can reuse it like:
```
- name: test
  uses: ./.github/actions/ci-e2e
 ```
 But this can be done on a separate PR

Reply:
this is a neat idea; my only concern is: the above is a workflow (snippet), not a proper action. I'm not sure how easy it is to wrap up these steps in a proper action.

Reply:
Let me do some homework and see